charts/minio/templates/deployment.yaml

```diff
@@ -69,6 +69,7 @@ spec:
         runAsUser: {{ .Values.securityContext.runAsUser }}
         runAsGroup: {{ .Values.securityContext.runAsGroup }}
         fsGroup: {{ .Values.securityContext.fsGroup }}
+        fsGroupChangePolicy: {{ .Values.securityContext.fsGroupChangePolicy }}
```

Comment:
Please try to contribute this change also to upstream. Otherwise this change would be overwritten by next update of the chart from the upstream.

Reply:
Done. [PR](https://github.com/minio/minio/pull/14528)